pkg/cloudprovider/providers/gce/gce_loadbalancer.go

```diff
@@ -120,7 +127,8 @@ func (gce *GCECloud) EnsureLoadBalancer(clusterName string, apiService *v1.Servi
 		return nil, err
 	}
 
-	loadBalancerName := cloudprovider.GetLoadBalancerName(apiService)
+	loadBalancerName := gce.GetLoadBalancerName(apiService)
```

Comment:
you can't update a load balancer name.  We have to accept the old name if it exists.